### DISCUSSION TITLE
Keep List Nested on Enter Click

### DISCUSSION THREAD

**Discussion by Ivcota** [vslinko/obsidian-outliner#135] [Archive]
It would be neat if double enter or single enter when the list is nested made a sibling bullet. 

Like this: 

https://user-images.githubusercontent.com/18635074/118522829-5be46380-b6f1-11eb-9dfd-ea333a9f372e.mp4

This is an example from version 1.1.5—I'm wondering if this is possible to reinstate?

Thanks!

**Comment by imed-ghomari** [comment]
yes this please, i'm on 1.3.1 and it still doesn't behave this way

**Reply by vslinko** [reply]
Moved it from Archive to Ideas.

**Reply by imed-ghomari** [reply]
Does this mean it'll be implemented in the future ?

**Reply by vslinko** [reply]
Yes, it could be implemented if I’ll have enough time.

**Comment by vslinko** [comment]
Should be fixed after 2.0 release

**Reply by imed-ghomari** [reply]
Thank you so much!
